src/Logic/Forms/RemoveTextForHI.cs

```diff
@@ -97,7 +97,7 @@ public string RemoveColon(string text)
             foreach (string line in lines)
             {
                 int indexOfColon = line.IndexOf(':');
-                if (indexOfColon > 0 && IsNotInsideBrackets(text, indexOfColon))
+                if (indexOfColon > 0 && IsNotInsideBrackets(line, indexOfColon))

```

Comment:
@niksedk you need to look at this.... isn't this a BUG?
`indexOfColon` as 'nothing' to do with `text`